requests/adapters.py

```diff
@@ -383,7 +384,8 @@ def proxy_headers(self, proxy):
 
         return headers
 
-    def send(self, request, stream=False, timeout=None, verify=True, cert=None, proxies=None):
+    def send(self, request, stream=False, timeout=None, verify=True, cert=None,
```

Comment:
I'm really not quite fond of changing this signature. Several people rely on this signature to write their own Adapters and if this is now required, it adds extra work for adapter authors.